core/scripts/cli/wallet.js

```diff
@@ -1,36 +1,98 @@
+// TODO:
+// Encrypt account data files
+// Show UMA token balance in wallet-info
+// Allow user to change directory in which wallets are stored
+// Allow more than one wallet backups
+// Connect to MetaMask
+
 const inquirer = require("inquirer");
+const os = require("os");
+
+// Wallet module helpers
+const createNewAccount = require("./wallet/createNewAccount");
+const createWalletDirectory = require("./wallet/createWalletDirectory");
+const loadSavedAccount = require("./wallet/loadSavedAccount");
+const restoreBackupAccount = require("./wallet/swapDefaultAndBackupAccounts");
+const readDefaultAccount = require("./wallet/readDefaultAccountInfo");
+const recoverAccount = require("./wallet/recoverAccount");
+const style = require("./textStyle");
+
+const ACTIONS = ["info", "init", "restore-backup", "recover", "help", "back"];
```

Comment:
nit: why not make this an object so it can be accessed like `ACTIONS.info`? Otherwise, `ACTIONS[0]` is pretty hard to read.

Reply:
good idea

Reply:
Done